pkgs/development/python-modules/crccheck/default.nix

```diff
@@ -1,18 +1,21 @@
-{ lib, stdenv, buildPythonPackage, fetchPypi
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy3k
 , nose }:
 
-buildPythonPackage rec {
+let
   pname = "crccheck";
-  version = "0.6";
+  version = "1.0";
+in buildPythonPackage {
+  inherit pname version;
 
   buildInputs = [ nose ];
```

Comment:
According to their setup.py https://sourceforge.net/p/crccheck/code/ci/default/tree/setup.py this is a checkInputs.